commitlint.config.js

```diff
@@ -2,7 +2,7 @@ module.exports = {
 	plugins: ['commitlint-plugin-jira-rules'],
 	extends: ['jira'],
 	rules: {
-		"header-max-length": [2, "always", 50],
+		"header-max-length": [2, "always", 80],
```

Comment:
Please open a new issue and new PR for this change.

Reply:
Yes, okay.